tests/modules/bowtie2/align/test.yml

```diff
@@ -1,83 +1,21 @@
-- name: bowtie2 align single-end
+- name: bowtie2 align test_bowtie2_align_single_end
   command: nextflow run ./tests/modules/bowtie2/align -entry test_bowtie2_align_single_end -c ./tests/config/nextflow.config -c ./tests/modules/bowtie2/align/nextflow.config
   tags:
-    - bowtie2
     - bowtie2/align
-  files:
-    - path: ./output/bowtie2/test.bam
-    - path: ./output/bowtie2/test.bowtie2.log
-    - path: ./output/bowtie2/bowtie2/genome.3.bt2
-      md5sum: 4ed93abba181d8dfab2e303e33114777
-    - path: ./output/bowtie2/bowtie2/genome.2.bt2
-      md5sum: 47b153cd1319abc88dda532462651fcf
-    - path: ./output/bowtie2/bowtie2/genome.1.bt2
-      md5sum: cbe3d0bbea55bc57c99b4bfa25b5fbdf
-    - path: ./output/bowtie2/bowtie2/genome.4.bt2
-      md5sum: c25be5f8b0378abf7a58c8a880b87626
-    - path: ./output/bowtie2/bowtie2/genome.rev.1.bt2
-      md5sum: 52be6950579598a990570fbcf5372184
-    - path: ./output/bowtie2/bowtie2/genome.rev.2.bt2
-      md5sum: e3b4ef343dea4dd571642010a7d09597
-
-- name: bowtie2 align paired-end
-  command: nextflow run ./tests/modules/bowtie2/align -entry test_bowtie2_align_paired_end -c ./tests/config/nextflow.config -c ./tests/modules/bowtie2/align/nextflow.config
-  tags:
     - bowtie2
-    - bowtie2/align
   files:
-    - path: ./output/bowtie2/test.bam
-    - path: ./output/bowtie2/test.bowtie2.log
-    - path: ./output/bowtie2/bowtie2/genome.3.bt2
-      md5sum: 4ed93abba181d8dfab2e303e33114777
-    - path: ./output/bowtie2/bowtie2/genome.2.bt2
-      md5sum: 47b153cd1319abc88dda532462651fcf
-    - path: ./output/bowtie2/bowtie2/genome.1.bt2
-      md5sum: cbe3d0bbea55bc57c99b4bfa25b5fbdf
-    - path: ./output/bowtie2/bowtie2/genome.4.bt2
-      md5sum: c25be5f8b0378abf7a58c8a880b87626
-    - path: ./output/bowtie2/bowtie2/genome.rev.1.bt2
-      md5sum: 52be6950579598a990570fbcf5372184
-    - path: ./output/bowtie2/bowtie2/genome.rev.2.bt2
-      md5sum: e3b4ef343dea4dd571642010a7d09597
+    - path: output/bowtie2/test.bam
+    - path: output/bowtie2/test.bowtie2.log
+      md5sum: 7b8a9e61b7646da1089b041333c41a87
+    - path: output/bowtie2/versions.yml
 
-- name: bowtie2 align single-end large-index
```

Comment:
Another one for you @chris-cheshire. You added these entries to `test.yml` for large indices but we don't have the corresponding entries in `main.nf` to actually test them? This is why all of these lines have automatically been deleted by `create-test-yml`.

Reply:
oh do we need to though? they use the same entry point, they just change a parameter

Reply:
yeah my bad will do